airflow/config_templates/config.yml

```
@@ -577,6 +577,78 @@
       type: string
       example: "connexion,sqlalchemy"
       default: ""
+- name: metrics
+  description: |
+    StatsD (https://github.com/etsy/statsd) integration settings.
+  options:
+    - name: statsd_on
+      description: |
+        Enables sending metrics to StatsD.
+      version_added: ~
+      type: string
+      example: ~
+      default: "False"
+    - name: statsd_host
+      description: ~
+      version_added: ~
+      type: string
+      example: ~
+      default: "localhost"
+    - name: statsd_port
+      description: ~
+      version_added: ~
+      type: string
+      example: ~
+      default: "8125"
+    - name: statsd_prefix
+      description: ~
+      version_added: ~
+      type: string
+      example: ~
+      default: "airflow"
+    - name: statsd_allow_list
+      description: |
+        If you want to avoid send all the available metrics to StatsD,
+        you can configure an allow list of prefixes to send only the metrics that
```

```suggestion
        you can configure an allow list of prefixes (comma separated) to send only the metrics that
```

A minor change to make it clearer to readers.